src/shred_collector/shred_receiver.zig

```diff
@@ -192,10 +211,34 @@ const REPAIR_RESPONSE_SERIALIZED_PING_BYTES = 132;
 const RepairPing = union(enum) { Ping: Ping };
 
 pub const ShredReceiverMetrics = struct {
-    invalid_repair_pings: *sig.prometheus.Counter,
+    received_count: *Counter,
```

Comment:
we use `num_` in `BlockstoreRpcApiMetrics` and here we use `_count` and in geyser we use `_total` and `_n` lol -- im cool with any of them but should be consistent

Reply:
I think `_count` and `_total` conform to the standard naming conventions, whereas `num_` does not. Grafana will complain if you don't use one of these.

I prefer "count" over "total" because I feel it is less ambiguous. "Count" clearly describes a number of discrete items or events, which is what we're typically using Counters for. I would expect anything called a "count" to be a positive integer. "Total" on the other hand is much more general, it could refer to a sum of any arbitrary numbers.

Reply:
im good with using `_count` everywhere

Reply:
Are you suggesting changing the metrics names now to all be consistent? My only concern here is breaking existing grafana dashboards.

Reply:
true - pushed it to a backlog issue - https://github.com/orgs/Syndica/projects/2/views/10?pane=issue&itemId=82929996